miniapp/CMakeLists.txt

```
@@ -29,43 +32,50 @@ if (DLAF_BUILD_TESTING)
   target_link_libraries(miniapp_triangular_solver PRIVATE DLAF_miniapp)
   target_add_warnings(miniapp_triangular_solver)
   DLAF_addPrecompiledHeaders(miniapp_triangular_solver)
+  install(TARGETS miniapp_triangular_solver RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
 endif()
 
 add_executable(miniapp_gen_to_std miniapp_gen_to_std.cpp)
 target_link_libraries(miniapp_gen_to_std PRIVATE DLAF)
 target_link_libraries(miniapp_gen_to_std PRIVATE DLAF_miniapp)
 target_add_warnings(miniapp_gen_to_std)
 DLAF_addPrecompiledHeaders(miniapp_gen_to_std)
+install(TARGETS miniapp_gen_to_std RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Should we do:
```suggestion
DLAF_addMiniapp(miniapp_gen_to_std ...)
```
I volunteer to do this in a separate PR...